nepal_compliance/nepali_num2words.py

```
@@ -0,0 +1,107 @@
+	Manual Nepali number to words conversion using Arba/Kharba system.
+	Handles numbers up to 10^11 (Kharba).
+	"""
+	if integer < 0:
+		return "Minus " + _nepali_in_words(-integer)
+
+	ones = ["", "One", "Two", "Three", "Four", "Five", "Six", "Seven", "Eight", "Nine"]
+	teens = ["Ten", "Eleven", "Twelve", "Thirteen", "Fourteen", "Fifteen", 
+	         "Sixteen", "Seventeen", "Eighteen", "Nineteen"]
+	tens = ["", "", "Twenty", "Thirty", "Forty", "Fifty", "Sixty", "Seventy", "Eighty", "Ninety"]
+	
+	def words_below_hundred(n):
```

Ruff (0.15.0)

[warning] Missing return type annotation for private function `words_below_hundred`

(ANN202)